src/ZombieDriver.php

```diff
@@ -164,15 +164,33 @@ public function back()
     public function setBasicAuth($user, $password)
     {
         if (false === $user) {
-            $this->server->evalJS("browser.authenticate().reset();stream.end();");
-
-            return;
+            $user = null;
+            $password = null;
         }
 
         $userEscaped = json_encode($user);
         $passwordEscaped = json_encode($password);
 
-        $this->server->evalJS("browser.authenticate().basic({$userEscaped}, {$passwordEscaped});stream.end();");
+        $js = <<<JS
+var username = $userEscaped;
```

Comment:
no. Both are valid. ``{ }`` allows to run embed arbitrary expression, but for simple variable access, it is not necessary.